test/java/seedu/addressbook/data/person/UniquePersonListTest.java

```diff
@@ -0,0 +1,71 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.assertTrue;
+import static seedu.addressbook.util.TestUtil.isIdentical;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.Tag;
+
+public class UniquePersonListTest {
+    private Tag tagPrizeWinner;
+    private Tag tagScientist;
+    private Tag tagMathematician;
+    private Tag tagEconomist;
+
+    private Person aliceBetsy;
+    private Person bobChaplin;
+    private Person charlieDouglas;
+    private Person davidElliot;
+
+    @Test
+    public void sort() throws Exception {

```

Comment:
I understand that you tried to align all the `=`s to make the code look cleaner but I am not sure if it is in accordance with the Coding Standards. In the document at https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#variables, they did align the `=`s under number 12. Associated constants. So I am unsure about this. Will get back to you. But I am glad that you put in the effort to make your code easy to read and clean.

Reply:
I was actually following the styling in addressbook-level2: https://github.com/nus-cs2103-AY1819S1/addressbook-level2/blob/master/test/java/seedu/addressbook/data/AddressBookTest.java

Reply:
Wow good job! We now have something concrete that we can bring up to Prof Damith. I'll ask him :)

Reply:
Good news :) Prof Damith said that we can allow it since both the code and the coding standard seems to hint at it.

Reply:
Okay, thanks!